### PR TITLE
Several fixes.

### PR DESCRIPTION
**Pull request by pozdnychev**

### CODE REVIEW

File: src/list.c [hunk 1]
```diff
@@ -8,13 +8,13 @@
 #include "list.h"
 
 /*
- * Allocate a new list_t. NULL on failure.
+ * Allocate a new tlist. NULL on failure.
  */
 
-list_t *
```
Comment (stephenmathieson) [inline src/list.c]: this is a breaking change, so no thanks

**Comment by stephenmathieson** [comment]
doesnt seem like anything is actually fixed here

we're not going to break the api here, so i'm going to close this pr

feel free to re-open with any non-breaking changes

**Comment by tj** [comment]
I always use `_t` :( haha, breaking all the rules. I prefer it because I usually have something like `user_t user` vs `user u` or similar

**Comment by pozdnychev** [comment]
@stephenmathieson Well, if it's a breaking change you could just change the major number.
@tj Lots of people are using _t, it's not wrong until you decide to share your code as a library/header, which may collide with POSIX.

Note: the same way identifiers starting with _ are reserved by POSIX (cf. http://pubs.opengroup.org/onlinepubs/9699919799/functions/V2_chap02.html)

**Pull request closed without merging**